.env.example

```diff
@@ -0,0 +1,3 @@
+OPENAI_API_KEY=
+SERPER_API_KEY=
```

Comment:
Tavily as apposed to Serper?

Reply:
Oops, initially I thought I will use serper here but then I changed it and forgot about this. Thanks!